cmd/revokeclearuser.go

```diff
@@ -74,11 +79,43 @@ func (p *ClearRevokeUserParams) PreInteractive(ctx ActionCtx) error {
 	if err = p.AccountContextParams.Edit(ctx); err != nil {
 		return err
 	}
-	if p.user == "" {
-		p.user, err = ctx.StoreCtx().PickUser(p.AccountContextParams.Name)
-		if err != nil {
-			return err
+
+	st := ctx.StoreCtx().Store
+	accClaim, err := st.ReadAccountClaim(p.AccountContextParams.Name)
+	if err != nil {
+		return err
+	} else if len(accClaim.Revocations) == 0 {
+		return nil
+	}
+	keyToUserName := map[string]string{}
+	if users, err := st.ListEntries(store.Accounts, p.AccountContextParams.Name, store.Users); err != nil {
```

Comment:
to me this reads better:
users, err := ...
if err != nil {
  return err
}

// no else required